2023-10-10 02:16:55,449 DEBUG    root            Loaded Command Group: ['gcloud', 'compute']
2023-10-10 02:16:55,451 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances']
2023-10-10 02:16:55,595 DEBUG    root            Loaded Command Group: ['gcloud', 'compute', 'instances', 'create']
2023-10-10 02:16:55,668 DEBUG    root            Running [gcloud.compute.instances.create] with arguments: [--boot-disk-size: "536870912000", --image-family: "tf-1-15-5", --image-project: "ml-images", --machine-type: "n1-highmem-8", --maintenance-policy: "TERMINATE", --restart-on-failure: "True", --zone: "asia-south1-c", INSTANCE_NAMES:1: "['samples']"]
2023-10-10 02:16:55,679 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 02:16:56,142 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/pegasus-401519/zones/asia-south1-c?alt=json HTTP/1.1" 200 None
2023-10-10 02:16:56,156 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 02:16:56,640 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "GET /compute/v1/projects/pegasus-401519?alt=json HTTP/1.1" 200 None
2023-10-10 02:16:56,655 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 02:17:00,436 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/pegasus-401519/zones/asia-south1-c/instances?alt=json HTTP/1.1" 200 None
2023-10-10 02:17:00,439 DEBUG    root            Operations to inspect: [(<Operation
 id: 7092441814435223595
 insertTime: '2023-10-09T13:47:00.290-07:00'
 kind: 'compute#operation'
 name: 'operation-1696884416780-6074eb0e319c5-c7736bce-22cbd798'
 operationType: 'insert'
 progress: 0
 selfLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-south1-c/operations/operation-1696884416780-6074eb0e319c5-c7736bce-22cbd798'
 startTime: '2023-10-09T13:47:00.291-07:00'
 status: StatusValueValuesEnum(RUNNING, 2)
 targetId: 109872606677162028
 targetLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-south1-c/instances/samples'
 user: 'hardiksharmah73@gmail.com'
 warnings: []
 zone: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-south1-c'>, 3)]
2023-10-10 02:17:00,442 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): compute.googleapis.com:443
2023-10-10 02:17:07,144 DEBUG    urllib3.connectionpool https://compute.googleapis.com:443 "POST /compute/v1/projects/pegasus-401519/zones/asia-south1-c/operations/operation-1696884416780-6074eb0e319c5-c7736bce-22cbd798/wait?alt=json HTTP/1.1" 200 None
2023-10-10 02:17:07,151 DEBUG    root            Operations to inspect: [(<Operation
 endTime: '2023-10-09T13:47:07.015-07:00'
 error: <ErrorValue
 errors: [<ErrorsValueListEntry
 code: 'ZONE_RESOURCE_POOL_EXHAUSTED'
 errorDetails: [<ErrorDetailsValueListEntry
 help: <Help
 links: [<HelpLink
 description: 'Troubleshooting documentation'
 url: 'https://cloud.google.com/compute/docs/resource-error'>]>>, <ErrorDetailsValueListEntry
 localizedMessage: <LocalizedMessage
 locale: 'en-US'
 message: 'A n1-highmem-8 VM instance is currently unavailable in the asia-south1-c zone. Alternatively, you can try your request again with a different VM hardware configuration or at a later time. For more information, see the troubleshooting documentation.'>>, <ErrorDetailsValueListEntry
 errorInfo: <ErrorInfo
 domain: 'compute.googleapis.com'
 metadatas: <MetadatasValue
 additionalProperties: [<AdditionalProperty
 key: 'attachment'
 value: ''>, <AdditionalProperty
 key: 'vmType'
 value: 'n1-highmem-8'>, <AdditionalProperty
 key: 'zone'
 value: 'asia-south1-c'>, <AdditionalProperty
 key: 'zonesAvailable'
 value: ''>]>
 reason: 'resource_availability'>>]
 message: "The zone 'projects/pegasus-401519/zones/asia-south1-c' does not have enough resources available to fulfill the request.  Try a different zone, or try again later.">]>
 httpErrorMessage: 'SERVICE UNAVAILABLE'
 httpErrorStatusCode: 503
 id: 7092441814435223595
 insertTime: '2023-10-09T13:47:00.290-07:00'
 kind: 'compute#operation'
 name: 'operation-1696884416780-6074eb0e319c5-c7736bce-22cbd798'
 operationType: 'insert'
 progress: 100
 selfLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-south1-c/operations/operation-1696884416780-6074eb0e319c5-c7736bce-22cbd798'
 startTime: '2023-10-09T13:47:00.291-07:00'
 status: StatusValueValuesEnum(DONE, 0)
 targetId: 109872606677162028
 targetLink: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-south1-c/instances/samples'
 user: 'hardiksharmah73@gmail.com'
 warnings: []
 zone: 'https://www.googleapis.com/compute/v1/projects/pegasus-401519/zones/asia-south1-c'>, 3)]
2023-10-10 02:17:07,151 DEBUG    root            Chosen display Format:yaml
2023-10-10 02:17:07,160 DEBUG    root            (gcloud.compute.instances.create) Could not fetch resource:
---
code: ZONE_RESOURCE_POOL_EXHAUSTED
errorDetails:
- help:
    links:
    - description: Troubleshooting documentation
      url: https://cloud.google.com/compute/docs/resource-error
- localizedMessage:
    locale: en-US
    message: A n1-highmem-8 VM instance is currently unavailable in the asia-south1-c
      zone. Alternatively, you can try your request again with a different VM hardware
      configuration or at a later time. For more information, see the troubleshooting
      documentation.
- errorInfo:
    domain: compute.googleapis.com
    metadatas:
      attachment: ''
      vmType: n1-highmem-8
      zone: asia-south1-c
      zonesAvailable: ''
    reason: resource_availability
message: The zone 'projects/pegasus-401519/zones/asia-south1-c' does not have enough
  resources available to fulfill the request.  Try a different zone, or try again
  later.
Traceback (most recent call last):
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 807, in Run
    resources = command_instance.Run(args)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/surface/compute/instances/create.py", line 699, in Run
    return compute_client.MakeRequests(requests)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 156, in MakeRequests
    utils.RaiseToolException(
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/utils.py", line 139, in RaiseToolException
    RaiseException(problems, calliope_exceptions.ToolException, error_message)
  File "/home/devilunraveled/GoogleCloud/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/utils.py", line 148, in RaiseException
    raise exception(
googlecloudsdk.calliope.exceptions.ToolException: Could not fetch resource:
---
code: ZONE_RESOURCE_POOL_EXHAUSTED
errorDetails:
- help:
    links:
    - description: Troubleshooting documentation
      url: https://cloud.google.com/compute/docs/resource-error
- localizedMessage:
    locale: en-US
    message: A n1-highmem-8 VM instance is currently unavailable in the asia-south1-c
      zone. Alternatively, you can try your request again with a different VM hardware
      configuration or at a later time. For more information, see the troubleshooting
      documentation.
- errorInfo:
    domain: compute.googleapis.com
    metadatas:
      attachment: ''
      vmType: n1-highmem-8
      zone: asia-south1-c
      zonesAvailable: ''
    reason: resource_availability
message: The zone 'projects/pegasus-401519/zones/asia-south1-c' does not have enough
  resources available to fulfill the request.  Try a different zone, or try again
  later.

2023-10-10 02:17:07,162 ERROR    root            (gcloud.compute.instances.create) Could not fetch resource:
---
code: ZONE_RESOURCE_POOL_EXHAUSTED
errorDetails:
- help:
    links:
    - description: Troubleshooting documentation
      url: https://cloud.google.com/compute/docs/resource-error
- localizedMessage:
    locale: en-US
    message: A n1-highmem-8 VM instance is currently unavailable in the asia-south1-c
      zone. Alternatively, you can try your request again with a different VM hardware
      configuration or at a later time. For more information, see the troubleshooting
      documentation.
- errorInfo:
    domain: compute.googleapis.com
    metadatas:
      attachment: ''
      vmType: n1-highmem-8
      zone: asia-south1-c
      zonesAvailable: ''
    reason: resource_availability
message: The zone 'projects/pegasus-401519/zones/asia-south1-c' does not have enough
  resources available to fulfill the request.  Try a different zone, or try again
  later.

